File: source/code/csharp/multitenancy/create_oasm_full_req.cs
var newMapping = client.Instantiate<IOrganizationAccountStoreMapping>()
    .SetAccountStore(existingDirectory)
    .SetListIndex(-1)
    .SetDefaultAccountStore(true)
    .SetDefaultGroupStore(true);

await bankOfAOrg.CreateAccountStoreMappingAsync(newMapping);

File: source/code/java/multitenancy/create_oasm_full_req.java
OrganizationAccountStoreMapping newMapping = client.instantiate(OrganizationAccountStoreMapping.class)
    .setAccountStore(existingDirectory)
    .setListIndex(-1)
    .setDefaultAccountStore(true)
    .setDefaultGroupStore(true);

bankOfAOrg.createAccountStoreMapping(newMapping);
